shapeworks_cloud/manage_workers.py

```diff
@@ -26,11 +27,14 @@ def inspect_queue(queue_name):
                 )
                 vhost = manager.user
             queue = manager.get_queue(vhost, queue_name)
-            num_messages = queue.get('messages_ready', num_messages)
+            num_messages = queue.get('messages', -1)
```

Comment:
Where are the API docs for `queue.get` and its possible parameters?

Reply:
These are the docs for the `pyrabbit` API Client, which include the `get_queue` function on the previous line: https://pyrabbit.readthedocs.io/en/latest/api.html#pyrabbit.api.Client.get_queue
Here, `queue` is a dict returned from that function. Here is an example of that dict:

<details><summary>Details</summary>
<p>

 ```
{
    "consumer_details": [
        {
            "arguments": {},
            "channel_details": {
                "connection_name": "172.19.0.1:58538 -> 172.19.0.5:5672",
                "name": "172.19.0.1:58538 -> 172.19.0.5:5672 (1)",
                "node": "rabbit@ff8af25c9090",
                "number": 1,
                "peer_host": "172.19.0.1",
                "peer_port": 58538,
                "user": "guest",
            },
            "ack_required": True,
            "active": True,
            "activity_status": "up",
            "consumer_tag": "None4",
            "exclusive": False,
            "prefetch_count": 0,
            "queue": {"name": "gpu", "vhost": "/"},
        }
    ],
    "arguments": {},
    "auto_delete": False,
    "backing_queue_status": {
        "avg_ack_egress_rate": 0.0,
        "avg_ack_ingress_rate": 0.0,
        "avg_egress_rate": 0.0,
        "avg_ingress_rate": 0.0,
        "delta": ["delta", 0, 0, 0, 0],
        "len": 0,
        "mode": "default",
        "next_seq_id": 0,
        "q1": 0,
        "q2": 0,
        "q3": 0,
        "q4": 0,
        "target_ram_count": "infinity",
    },
    "consumer_capacity": 1.0,
    "consumer_utilisation": 1.0,
    "consumers": 1,
    "deliveries": [],
    "durable": True,
    "effective_policy_definition": {},
    "exclusive": False,
    "exclusive_consumer_tag": None,
    "garbage_collection": {
        "fullsweep_after": 65535,
        "max_heap_size": 0,
        "min_bin_vheap_size": 46422,
        "min_heap_size": 233,
        "minor_gcs": 16,
    },
    "head_message_timestamp": None,
    "idle_since": "2024-04-10 15:25:44",
    "incoming": [],
    "memory": 17240,
    "message_bytes": 0,
    "message_bytes_paged_out": 0,
    "message_bytes_persistent": 0,
    "message_bytes_ram": 0,
    "message_bytes_ready": 0,
    "message_bytes_unacknowledged": 0,
    "messages": 0,
    "messages_details": {"rate": 0.0},
    "messages_paged_out": 0,
    "messages_persistent": 0,
    "messages_ram": 0,
    "messages_ready": 0,
    "messages_ready_details": {"rate": 0.0},
    "messages_ready_ram": 0,
    "messages_unacknowledged": 0,
    "messages_unacknowledged_details": {"rate": 0.0},
    "messages_unacknowledged_ram": 0,
    "name": "gpu",
    "node": "rabbit@ff8af25c9090",
    "operator_policy": None,
    "policy": None,
    "recoverable_slaves": None,
    "reductions": 10225,
    "reductions_details": {"rate": 0.0},
    "single_active_consumer_tag": None,
    "state": "running",
    "type": "classic",
    "vhost": "/",
}

 ```

</p>
</details>

Reply:
Thanks.